neo4jComRedirects/redirects-labs.conf

```diff
@@ -201,6 +201,12 @@ rewrite ^/developer/spark/current/? /docs/spark/current/ permanent;
 rewrite ^/developer/spark/4\.0/? /docs/spark/4.0/ permanent;
 rewrite ^/developer/spark/4\.1/? /docs/spark/4.1/ permanent;
 
+# Redirects following the move of Kafka Connector documentation from /developer to /docs (October 2022):
+rewrite ^/labs/kafka/? /docs/kafka/ permanent;
+rewrite ^/labs/kafka/current/? /docs/kafka/ permanent;
+rewrite ^/labs/kafka/4\.0/? /docs/kafka/ permanent;
+rewrite ^/labs/kafka/4\.1/? /docs/kafka/ permanent;
```

Comment:
As @recrwplay observed, these lines can potentially be skipped as the `rewrite ^/labs/kafka/? /docs/kafka/` line would already redirect any page (plus we redirect all pages to the same `/docs/kafka/` home page).

Reply:
Updated this following @adam-cowley's suggestion.